test/conformance/helpers/broker_control_plane_test_helper.go

```diff
@@ -39,7 +39,7 @@ import (
 func BrokerV1Beta1ControlPlaneTestHelperWithChannelTestRunner(
 	t *testing.T,
 	brokerClass string,
-	channelTestRunner lib.ChannelTestRunner,
+	channelTestRunner lib.ComponentsTestRunner,
```

Comment:
wanna change this great (IMO) rename here as well:

* https://github.com/knative/eventing-contrib/blob/master/test/conformance/main_test.go#L30

Reply:
yep, will do